src/App.js

```diff
@@ -1,25 +1,57 @@
 import React from 'react';
+import { Clock } from './Clock';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+export class App extends React.Component {
+  state = {
+    clockName: 0,
+    isClockVisible: true,
+  };
 
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
+  render() {
+    const { isClockVisible, clockName } = this.state;
 
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+    return (
+      <div className="App">
+        <h1>A Clock</h1>
+        {isClockVisible && <Clock name={clockName} />}
+        <div>
+          <button
+            className="App__button"
+            type="button"
+            onClick={() => {
+              this.setState({ isClockVisible: true });
+            }}
```

Comment:
It's a good practice to create separate methods(functions) for certain operations.
E.g. create method 
```
showClock = () => {
    this.setState({
      isClockVisible: true,
    });
  }
```

ant then just call it here:

`onClick={this.showClock}`

It will make the code more readable and allow you to reuse this function